targets/targets.json

```diff
@@ -3421,7 +3421,8 @@
             "SERIAL_ASYNCH",
             "TRNG",
             "FLASH",
-            "MPU"
+            "MPU",
+            "USBDEVICE"
```

Comment:
Can't be part of PR, as board doesn't have any USB connection.
This part has to be part of mbed_app.json